#request portion containing 4 int64 discrete values
int64 id
int64 object_size
int64 human_action
int64 human_expression
---
#response portion containing 3 float values
float64 p_happy
float64 p_sad
float64 p_neutral
